server/routes/api/article.js

```diff
@@ -11,7 +11,7 @@ const router = express.Router();
 router.post('/articles', checkToken, upload.single('imgUrl'), ArticleValidations.validateCreateArticle, ArticleController.createArticle);
 router.get('/articles/feed', ArticleController.getAllArticles);
 router.get('/articles', checkToken, ArticleController.getUserArticles);
-router.get('/articles/:slug', checkToken, ArticleController.getSingleArticle);
```

Comment:
I remove the authorization check on this endpoint because it won't be protected. A user that has not register can still read articles on the site.